Anlab.Mvc/Controllers/AdminController.cs

```diff
@@ -97,6 +97,8 @@ public IActionResult EditUser(string id, [Bind("FirstName,LastName,Name,Phone,Ac
         }
 
         [Authorize(Roles = RoleCodes.Admin)]
+        [HttpPost]
+        [ValidateAntiForgeryToken]
```

Comment:
@jSylvestre ah, yea i wouldn't have noticed except the attribute was added in this PR.